app/src/lib/stores/app-store.ts

```diff
@@ -829,6 +829,7 @@ export class AppStore extends TypedBaseStore<IAppState> {
           aheadBehindOfInferredBranch.behind
       ) {
         this._setDivergingBranchBannerVisibility(true)
+        this._recordDivergingBranchBannerDisplayed()
```

Comment:
This might erroneously get incremented for `true` -> `true` transitions which we handle inside `_setDivergingBranchBannerVisibility`. Can we move this inside `_setDivergingBranchBannerVisibility`?

Reply:
Yea, that seems like a much better place to put it. 👍